src/settings/mod.rs

```diff
@@ -65,8 +65,10 @@ deserialize_and_validate! {
     (BaseUri, base_uri, "base URI"),
     /// Host name or IP address type.
     (Host, host, "host name or IP address"),
+    /// Logging level type.
+    (LoggingLevel, logging_level, "'normal', 'debug'm 'critical' or 'off'"),
```

Comment:
> 'debug'm

Is that `'debug',`?

(I was going to comment on "why not call this 'info', the traditional naming", but then I see that Rocket interally has already made this choice and calls it `'normal'`. So, when in Rome...)